src/frontend/get-github-setup.ts

```diff
@@ -1,22 +1,33 @@
 import { jiraDomainOptions } from "./validations";
 import { Request, Response } from "express";
-import { getJiraMarketplaceUrl } from '../util/getUrl';
+import {
+	getJiraMarketplaceUrl,
+	getGitHubConfigurationUrl,
+} from "../util/getUrl";
 
 /*
-When this request is made: Installing from Jira Marketplace - GitHub org does not have Jira installed.
-Redirects users back to github/configuration to install their Jira instance in GitHub org/s.
-If the installation was done from Jira Marketplace, the app is already installed.
+Handles redirects for both the installation flow from Jira and
+the installation flow from GH.
+- From Jira: user has already installed the app and is redirected to the connect an org pg
+- From GH:
+		- If we have the users Jira host, redirect to marketplace.
+		- Otherwise, render the setup page.
 */
 export default (req: Request, res: Response): void => {
 	req.log.info("Received get github setup page request");
 
-	if (req.session.jiraHost) {
+	if (req.headers.referer) {
```

Comment:
This property is only present when we install the app via the Jira flow. Not sure if this is the best way to determine if someone is coming from the Jira or GH side but open to suggestions for distinguishing between the two.

Reply:
I'm not familiar with these changes, but random idea: could we maybe use different URLs in Jira and in GitHub (e.g. add a query parameter)?

Reply:
Question: why are we not feature-flagging this change?

Reply:
Answering to myself, probably because this is some code-shuffle refactoring :)